scripts/perftest/python/utils_exec.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+import subprocess
+import shlex
+import re
+
+# Subprocess and log parsing related functions
+
+
+# TODO
+# return_code and better error handling
+def subprocess_exec(cmd_string, extract=None):
+    """
+    Execute the input string as subprocess
+
+    cmd_string: String
+    Input string to be executed as a sub process
+
+    extract: String
+    Based on this we extract logs accordingly
+
+    return: String
+    Based on extract we return the relevant string
+    """
+    # Debug
+    # print(cmd_string)
+    proc1 = subprocess.Popen(shlex.split(cmd_string), stdout=subprocess.PIPE,
+                             stderr=subprocess.PIPE)
+
+    error_arr, out_arr = get_std_out(proc1)
+    std_outs = out_arr + error_arr
+    return_data = proc1.returncode
+
+    if extract == 'Time':
+        return_data = parse_time(std_outs)
+    if extract == 'dir':
+        return_data = parse_dir(std_outs)
+
+    return return_data
+
+
+def get_std_out(process):
```

Comment:
This name is somewhat misleading, since you return both - `stdout` and `stderr`.  Maybe you could rename it to something more appropriate?